core/src/replay_stage.rs

```diff
@@ -139,6 +139,7 @@ pub struct ReplayStageConfig {
     pub wait_for_vote_to_start_leader: bool,
     pub ancestor_hashes_replay_update_sender: AncestorHashesReplayUpdateSender,
     pub tower_storage: Arc<dyn TowerStorage>,
+    pub wait_to_vote_slot: Option<Slot>,
```

Comment:
Maybe add a comment here, what is the purpose of this field and when/where it should be used.